docs/package.json

```diff
@@ -67,6 +68,9 @@
 		"rimraf": "^5.0.1",
 		"start-server-and-test": "^2.0.0"
 	},
+	"devDependencies": {
+		"shx": "^0.3.4"
+	},
```

Comment:
Just for my own knowledge, is this standard practice for this project, for FF or just in general?

Reply:
It's sort of a best practice for all workspace root projects. Practically speaking it doesn't make much difference since they're not real packages that are published. But since it doesn't matter, it's easiest to keep everything in either deps or devdeps rather than splitting them up. Only applies to workspace root projects.